docker/cuda-tf/Dockerfile

```diff
@@ -72,6 +72,7 @@ ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:/usr/local/cuda/lib64:$LD
 # Link the libcuda stub to the location where tensorflow is searching for it and reconfigure
 # dynamic linker run-time bindings
 RUN ln -s /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/libcuda.so.1 \
+    && ln -s /usr/local/cuda-11.0/targets/x86_64-linux/lib/libcusolver.so.10 /usr/local/cuda-11.0/targets/x86_64-linux/lib/libcusolver.so.11 \
```

Comment:
I'm gonna remember this trick the next time I need to upgrade a library 😆

Reply:
Ah nvm, `libcusolver.so.11` can be installed by doing `apt-get install libcusolver-11-2`. But for cuda version 11.0 we only install ` libcusolver-11-0` which only comes with ` libcusolver.so.10`. I am preparing another PR actually that will create images with cuda only and then with cuda and tensorflow. I will close this and then just open that larger PR. I will install newer libraries in there that provide these packages.